api/v1beta1/cryostat_types.go

```diff
@@ -342,13 +342,6 @@ type NetworkConfigurationList struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	CoreConfig *NetworkConfiguration `json:"coreConfig,omitempty"`
-	// Specifications for how to expose the Cryostat command service,
-	// which serves the WebSocket command channel.
-	// +optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
-	//
-	// Deprecated: CommandConfig is no longer used.
-	CommandConfig *NetworkConfiguration `json:"commandConfig,omitempty"`
```

Comment:
I would leave this in v1beta1 for API compatibility. We'll just drop the value when converting to v1beta2 as you've already done.